git-ssh.md

```diff
@@ -0,0 +1,75 @@
+# Installing SSH Key on Macbook for GitHub access
+
+I'm not going to walk through the reasoning behind using SSH keys as authorization or why GitHub is making this change. Let's just dive into the process and get the SSH keys created, installed, and referenced on GitHub.
+
+## Create the SSH Key
+1. Open terminal. **I recommend using the terminal application on your MacBook rather than the integrated terminal in VS Code.**
```

Comment:
> **I recommend using the terminal application on your MacBook rather than the integrated terminal in VS Code.**

Cut this -- I'm guessing this doc will be shared during T1 (week 1), so students won't know what VSCode is.